src/settings/MappingProfiles/MappingProfileDetails/MappingProfileDetails.js

```diff
@@ -0,0 +1,190 @@
+                        </Row>
+                        <Row>
+                          <Col xs>
+                            <KeyValue
+                              data-test-mapping-profile-folio-record-type
+                              label={<FormattedMessage id="stripes-data-transfer-components.folioRecordType" />}
+                            >
+                              <List
+                                items={record.recordTypes}
+                                itemFormatter={recordType => (
+                                  <FormattedMessage
+                                    id={FOLIO_RECORD_TYPES[recordType].captionId}
+                                    tagName="li"
+                                  />
+                                )}
+                                isEmptyMessage={<NoValue />}
+                                listClass={css.recordTypesList}
+                              />
+                            </KeyValue>
+                          </Col>
+                        </Row>
+                        <Row>
+                          <Col xs>
+                            <KeyValue
+                              data-test-mapping-profile-output-format
+                              label={<FormattedMessage id="ui-data-export.outputFormat" />}
+                              value={record.outputFormat}
+                            />
+                          </Col>
+                        </Row>
+                        <Row>
+                          <Col xs>
+                            <KeyValue
+                              data-test-mapping-profile-description
+                              label={<FormattedMessage id="ui-data-export.description" />}
+                              value={record.description || <NoValue />}
+                            />
+                          </Col>
+                        </Row>
+                      </Accordion>
+                      <Accordion label={<FormattedMessage id="ui-data-export.transformations" />}>
+                        <MultiColumnList
+                          id="mapping-profile-details-transformations"
+                          contentData={record.transformations}
+                          columnMapping={columnMapping}
+                          columnWidths={columnWidths}
+                          visibleColumns={visibleColumns}
+                          formatter={formatter}
+                          isEmptyMessage={<FormattedMessage id="ui-data-export.mappingProfiles.transformations.emptyMessage" />}
+                        />
+                      </Accordion>
+                    </AccordionSet>
+                  </AccordionStatus>
+                </div>
+              )
+            }
+          </FullScreenView>
+        </Layer>
+      )}
+    </FormattedMessage>
+  );
+};
+
+MappingProfileDetails.propTypes = {
+  onCancel: PropTypes.func,
+  mappingProfile: PropTypes.object,
```

Comment:
Could you please use `.shape` to describe proptypes more detailed?

Reply:
Done, please check it